fog/ingest/enclave/trusted/Cargo.lock

```diff
@@ -1279,6 +1279,7 @@ dependencies = [
  "hkdf",
  "lazy_static",
  "mc-account-keys",
+ "mc-attest-core",
```

Comment:
Oof, this is pretty undesirable.

Reply:
OK, I'll duplicate `VerificationReport` and remove this dep